faq/audit-trail.mdx

```diff
@@ -0,0 +1,29 @@
+---
+meta:
+  title: Audit Trail FAQ
+  description: Get answers on using Scaleway's Audit Trail to monitor events, enhance security, and track activities across your resources
+content:
+  h1: Audit Trail
+hero: assets/iam.webp
+dates:
+  validation: 2024-11-26
+category: identity-and-access-management
+---
+
+## What is Audit Trail?
+
+Audit Trail is a tool that holds a record of events and changes performed within a Scaleway Organization.
+
+## Why do I need Audit Trail?
+
+Audit Trail helps you ensure accountability and security by recording who did what and when within your Scaleway Organization. It tracks activity across all your resources (identity of the principal, date of activity, source IP address, API method used, and status of the request).
+
+You can use Audit Trail for troubleshooting, compliance verification, and analysis in the event of a breach.
+
+## How am I billed for using Audit Trail?
+
+Audit Trail is free of charge during Beta.
+
```

Comment:
We could add a question that covers the fact that Audit Trail itself is regionalized, and that events in a given region are saved in the same region on Audit Trail